AI/Nullkiller/Engine/Nullkiller.cpp

```diff
@@ -221,6 +222,7 @@ void Nullkiller::makeTurn()
 	boost::lock_guard<boost::mutex> sharedStorageLock(AISharedStorage::locker);
 
 	const int MAX_DEPTH = 10;
+	const int FAST_TASK_MINIMAL_PRIORITY = 0.7;
```

Comment:
Build failed because of that line, surely not what you intended.

Reply:
Yes, for sure.

Reply:
@nullkiller maybe use constexpr int?